sklearn/impute/tests/test_impute.py

```diff
@@ -445,6 +445,17 @@ def test_imputation_constant_pandas(dtype):
     assert_array_equal(X_trans, X_true)
 
 
+@pytest.mark.parametrize('imputer', (SimpleImputer, IterativeImputer))
+def test_imputation_missing_value_in_test_array(imputer):
+    # [Non Regression Test] Missing value in test set should not throw an error
+    # and return a finite dataset
+    train = [[1], [2]]
+    test = [[3], [np.nan]]
+    imputer = imputer(add_indicator=True)
+    imputer.fit(train).transform(test)
+
+
+@pytest.mark.filterwarnings('ignore: The default value of cv')  # 0.22
```

Comment:
revert?

Reply:
Could you explain what changes need to be made here? I went through local history with the assumption that I may have accidentally made some formatting changes (didn't see any). I think the test still applies since my behavior doesn't touch the scope of the test that you are suggesting to revert.

Reply:
You need to remove this line. We removed this deprecation message in some earlier commit.